frontend-opal/App.js

```diff
@@ -1,7 +1,14 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
```

Comment:
@martinupset this file shouldnt be changed

Reply:
ok